### ISSUE TITLE
[Bug] Broken channel view 

### ISSUE THREAD

**Issue by github-throwaway** [ram-on/SkyTube#64]
Trying to access this channel ([**NEO MAGAZIN ROYALE**](https://www.youtube.com/channel/UCNNEMxGKV1LsKZRt4vaIbvw)) I get a really interesting bug.

| ![screenshot_2017-02-03-19-35-37](https://cloud.githubusercontent.com/assets/25488086/22620035/17a74c78-eb03-11e6-8977-0fc2999bad74.png) | ![screenshot_2017-02-03-19-35-47](https://cloud.githubusercontent.com/assets/25488086/22620036/17a93a60-eb03-11e6-955c-661894be80b3.png) | ![screenshot_2017-02-03-19-35-56](https://cloud.githubusercontent.com/assets/25488086/22620037/17a95b80-eb03-11e6-8f8f-015b800417f1.png) | ![screenshot_2017-02-03-19-36-04](https://cloud.githubusercontent.com/assets/25488086/22620038/17a99942-eb03-11e6-940e-9869f1d3d4a8.png) |
|:---:|:---:|:---:|:---:|

First there are no videos on the channel, if you then do a pull-to-refresh you get one four month old video, if you then do another pull-to-refresh you get a different video which is ten month old, doing a third pull-to-refresh you end up with no video again. Trying to refresh again doesn't change anything you are left with no video, unless you go back to the start page and load the channel view and start this whole process all over again.

**N9005 | Android 5.0 | SkyTube 2.2**

**Comment by xamica** [comment]
I have same issue too. But refresh option don't help me anyway, I just get empty channel.

**Comment by github-throwaway** [comment]
Interestingly enough the notifications still work for that channel.

**Comment by atomjack** [comment]
@ram-on I got this as well (no videos showing up at all, though), although it looks like it's due to the channel having its language set to German. Once I checked the box for German in the Preferred Language option, the videos showed up just fine. 

I noticed [here](https://github.com/ram-on/SkyTube/blob/master/app/src/main/java/free/rm/skytube/businessobjects/YouTubeVideo.java#L289) it's filtering out any languages that are not checked in the preferred language option. I'm wondering if maybe the better way to approach it would be to by default have it not restrict any, unless you want it to only show certain languages?
